gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -765,13 +768,13 @@ public static SignUrlOption withMd5() {
     }
 
     /**
-     * Service account credentials which are used for signing the URL.
-     * If not provided an attempt will be made to get it from the environment.
+     * Signing credentials which should be used to sign the URL. If not provided an attempt will be
+     * made to get it from the environment.
      *
      * @see <a href="https://cloud.google.com/storage/docs/authentication#service_accounts">Service
      *     account</a>
      */
-    public static SignUrlOption serviceAccount(ServiceAccountAuthCredentials credentials) {
+    public static SignUrlOption serviceAccount(SigningAuthCredentials credentials) {
```

Comment:
Java doc may change per my earlier suggestion.

Also, I think method name should change.